riotfile.py

```diff
@@ -162,14 +148,14 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             command="mypy {cmdargs}",
             create=True,
             pkgs={
-                "mypy": latest,
-                "envier": latest,
-                "types-attrs": latest,
-                "types-docutils": latest,
-                "types-protobuf": latest,
-                "types-PyYAML": latest,
-                "types-setuptools": latest,
-                "types-six": latest,
+                "mypy": "==0.991",
+                "envier": "==0.4.0",
+                "types-attrs": "==19.1.0",
+                "types-docutils": "==0.19.1.1",
+                "types-protobuf": "==3.20.4.5",
+                "types-PyYAML": "==6.0.12.2",
+                "types-setuptools": "==65.6.0.0",
+                "types-six": "==1.16.21.4",
```

Comment:
we don't want any of these as "latest"?

Reply:
I kept them at their previously-pinned versions from dependencies.py. We can make them track `latest` in a subsequent pull request.

Reply:
ok, I am ok with updating them to latest in follow-up PR.

probably safer to pin now.